run.sh

```diff
@@ -8,3 +8,8 @@ set -e
 if [ "${1}" = "refseq" ] ; then
   python -m src.refdata_sync.refseq_sync $2
 fi
+
+if [ "${1}"= "ebi_metagenomes"] ; then
```

Comment:
does `=` need a space around it? i assume not but i never know w bash

Reply:
see below